hack/observability/capi-state-metrics/kustomization.yaml

```diff
@@ -0,0 +1,53 @@
+resources:
```

Comment:
Why do we have to rename?

Reply:
We don't have to, I for now just did because it is limitted to the CAPI CR metrics for now. the other `kube_` metrics won't get exposed for now (and thus save memory etc.).

Reply:
But I assume we can eventually just use one kube-state-metrics directly from the chart with kube_ + our metrics?

Reply:
Yes it is possible.

Reply:
I would prefer one kube-state-metrics with all metrics as the end-state of this PR (I realize it's WIP and you're using it to upstream stuff :))

Reply:
👍